cfy_manager/components/validations.py

```diff
@@ -360,11 +386,25 @@ def _check_internal_ca_cert():
 
 def _validate_cert_inputs():
     _check_internal_ca_cert()
-    _check_key_and_cert('internal')
-    _check_key_and_cert('postgresql_server')
-    _check_key_and_cert('postgresql_client')
-    _check_key_and_cert('external')
-    _check_key_and_cert('external_ca')
+    for component in (
```

Comment:
maybe instead of component name it `ssl_input` since we don't have `internal`,`external` or `external_ca` components and it can be misleading.

Reply:
Changed.